web/template/layout/insert-tabs.gohtml

```diff
@@ -0,0 +1,64 @@
+{{ define "insert-tabs" }}
+    <div class="govuk-tabs" data-module="govuk-tabs">
+        <h2 class="govuk-tabs__title">Inserts</h2>
+        <ul class="govuk-tabs__list">
+            {{ range .DocumentInsertKeys }}
+                <li class="govuk-tabs__list-item" data-module="tab-link" id="{{ . }}">
+                    <a class="govuk-tabs__tab" href="#{{ . }}">
+                        {{ capitalise . }}
+                    </a>
+                </li>
+            {{ end }}
+        </ul>
+
+        {{ range $k := .DocumentInsertKeys }}
+            <div class="govuk-tabs__panel" data-module="tab-content" id="{{ $k }}">
+                <table class="govuk-table my-table table__no-border">
+                    <tbody class="govuk-table__body">
+                    {{ range $t := $.DocumentInsertTypes }}
+                        {{if eq $t.Key $k }}
+                            <tr class="govuk-table__row">
+                                <td class="govuk-table__cell">
+                                    <div class="govuk-checkboxes" data-module="govuk-checkboxes">
+                                        <div class="govuk-checkboxes__item">
+                                            <input class="govuk-checkboxes__input"
+                                                   id="f-{{ $t.Handle }}"
```

Comment:
because you can have the same handle in multiple tabs you can end up with an id being reused on the same page, could these be prefixed with the tab too?